### ISSUE TITLE
Menu component with active sub-item missing `aria-activedescendant` 

### ISSUE THREAD

**Issue by jhung** [platform-coop-toolkit/pinecone#173]
**Describe the bug**
`aria-activedescendant` property is missing from the parent menu item if child item is active (i.e. has `aria-current=page`).

**To Reproduce**
1. Load the Menu component.
2. Examine source code for `Menu Item Four` (the menu item with the active sub-tem).
3. Notice there is no `aria-activedescendant` property.

**Expected behavior**
The menu with active child should have `aria-descendant` property with the value of the id of the active sub-item.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox
 - Version  72.0.1

**Additional context**
See:
* https://www.w3.org/TR/wai-aria-practices/#kbd_focus_activedescendant
* https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-activedescendant_attribute

**Comment by jhung** [comment]
Just a note that the Language menu will need an active descendant as well.

**Comment by greatislander** [comment]
@jhung To clarify, what element should the `aria-activedescendant` attribute be attached to? The list item or the button which controls the submenu?
